### PR TITLE
feat: add git username and token to script parameters

### PR DESCRIPTION
**Pull request by vsukhin**
## Pull request description 

Add git username and token for repository model

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

- New fields for Repository model - git username and token

## Fixes

-

### CODE REVIEW

**Review by jasmingacic — COMMENTED** [review]
Where are you going to store these values once execution has completed? Make sure this gets obfuscated

**Comment by vsukhin** [comment]
They will be stored in secrets and read from them at pod creation
https://github.com/kubeshop/testkube/pull/786/files#diff-5110c5aeb9333fdd2dc9740111cfc8767d0d0107bf8d5d7ff98fe631fb48db9dR113
https://github.com/kubeshop/testkube/pull/786/files#diff-659e3bf73f85b88b2a79c592d6292a33da1d5234b2255064823f3d1c67782de9R389

**Comment by jasmingacic** [comment]
looks like one way of doing it. I don't think it is ideal solution but let's go with it

**Review by jasmingacic — APPROVED** [review]

**Comment by vsukhin** [comment]
Yes, we can start using Vault at some point, if we need to have secure storage

File: apis/script/v1/script_types.go [hunk 1]
```diff
@@ -42,14 +42,18 @@ type ScriptSpec struct {
...
+	// Type_ repository type
 	Type_ string `json:"type"`
-	// uri of content file or git directory
+	// Uri of content file or git directory
 	Uri string `json:"uri"`
 	// branch/tag name for checkout
 	Branch string `json:"branch"`
 	// if needed we can checkout particular path (dir or file) in case of BIG/mono repositories
 	Path string `json:"path,omitempty"`
+	// git auth username for private repositories
```
Comment (exu) [inline apis/script/v1/script_types.go:53]: I'm a little afraid about obfuscation of this data, shouldn't we simply pass reference to a secret ? 
what do you think guys?

Comment (exu) [reply apis/script/v1/script_types.go:53]: just looked at create - it need to be here :) ignore my last comment - mapper need to be changed to covert it - and probably is already :)

**Review by exu — APPROVED** [review]

**Pull request merged**